.travis.yml

```diff
@@ -13,6 +13,8 @@ env:
   - TOXENV=py2-docs
 #  - TOXENV=py3-docs
   - TOXENV=py2-cover,py3-cover,coverage
+  - TOXENV=py27-pyramid14
+  - TOXENV=py27-pyramid15
```

Comment:
Nevermind, I assume it's because tox doesn't install the alpha pyramid with pip. It's not a big deal.